src/transformers/models/tapas/configuration_tapas.py

```diff
@@ -12,7 +12,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-""" TAPAS configuration. Adds additional hyperparameters to the configuration of BERT."""
+"""
+ TAPAS configuration. Adds additional hyperparameters to the configuration of BERT.
+
+Hyperparameters are taken from run_task_main.py and hparam_utils.py of the original implementation. URLS: -
+https://github.com/google-research/tapas/blob/master/tapas/run_task_main.py -
+https://github.com/google-research/tapas/blob/master/tapas/utils/hparam_utils.py
```

Comment:
List was not properly formatted so got all weird.
```suggestion
Hyperparameters are taken from run_task_main.py and hparam_utils.py of the original implementation. URLS:

- https://github.com/google-research/tapas/blob/master/tapas/run_task_main.py
- https://github.com/google-research/tapas/blob/master/tapas/utils/hparam_utils.py
```